src/offset/utc.rs

```diff
@@ -40,9 +40,14 @@ use crate::{Date, DateTime};
 /// assert_eq!(Utc.timestamp_opt(61, 0).unwrap(), dt);
 /// assert_eq!(Utc.with_ymd_and_hms(1970, 1, 1, 0, 1, 1).unwrap(), dt);
 /// ```
-#[derive(Copy, Clone, PartialEq, Eq, Hash)]
-#[cfg_attr(feature = "rkyv", derive(Archive, Deserialize, Serialize))]
-#[cfg_attr(feature = "rkyv", archive_attr(derive(Clone, Copy, PartialEq, Eq, Debug, Hash)))]
+#[derive(Clone, Copy, Eq, Hash, PartialEq)]
```

Comment:
I sorted the derives alphabetically to better compre against the `archive_attr` equivalents.